package.json

```diff
@@ -223,6 +219,7 @@
     "typings-for-css-modules-loader": "^1.7.0",
     "uglifyjs-webpack-plugin": "^2.2.0",
     "webdriverio": "^5.11.6",
+    "webpack": "^4.36.1",
```

Comment:
how about using latest 4.43.0. 
moving this do devDependency might do the slug diff staff 👍

Reply:
it is already in devDependency.  That is what this PR does.  I'll look into upgrading it.

Reply:
We are already getting 4.43.0 by default:

> $ npm list webpack
alchemy-client@0.10.4 [...]alchemy
+-- @storybook/react@5.3.18
| +-- @storybook/core@5.3.18
| | +-- corejs-upgrade-webpack-plugin@2.2.0
| | | `-- webpack@4.43.0  deduped
| | `-- webpack@4.43.0  deduped
| `-- webpack@4.43.0  deduped
`-- webpack@4.43.0

Reply:
yes because it is with ^ ..
please set that explicitly : `"webpack": "4.43.0"`

Reply:
I am skeptical that moving this into devDependencies really did the trick.  It wasn't working last time I tried.  May justr be the usual unpredictable Heroku behavior.

The difference this time is that I rebased on the latest dev and rebuilt package-lock.json.  Now I see something which I _think_ has changed here:

> alchemy-client@0.10.4 C:\Users\dkent\Documents\Projects\GitHub\DAOStack\alchemy
`-- 3box@1.17.1
  `-- ipfs@0.40.0 
    `-- UNMET DEPENDENCY ipfs@0.38.1
>
> npm ERR! missing: ipfs@0.38.1, required by ipfs@0.40.0
npm ERR! extraneous: ipfs@0.38.1 C:\Users\dkent\Documents\Projects\GitHub\DAOStack\alchemy\node_modules\ipfs\node_modules\ipfs

I don't remember it saying "UNMET DEPENDENCY ipfs@0.38.1" before.  ipfs@0.38.1 in fact does appear to be missing now under ipfs/node_modules.  I further confirm the size of IPFS in the slug has vastly diminished: 

![image](https://user-images.githubusercontent.com/1821666/79978863-9936dd00-846e-11ea-89d5-f9fef391b925.png)